.gitignore

```diff
@@ -3,5 +3,6 @@
 
 /node_modules/
 /vendor/
+
```

Comment:
@erikyo did you mean to add a line here?

Reply:
no... however it goes, let's say a collateral damage of the lint. I would leave it since, theoretically, the empty end line is required at the end of each document

Reply:
Makes sense to me, I agree empty lines between groups is standard.

Reply:
but you are right maybe a comment would be nice to separate things (for example, I hadn't noticed that he had a line underneath)

Reply:
I've started to make some updates to this in https://github.com/WordPress/wp-feature-notifications/pull/240